oojs.js

```diff
@@ -3,10 +3,28 @@
   // A songs property that is determined by some input. It should contain multiple songs, each of which can just be a song title.
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
+class Playlist{
+  constructor(title){
+    this.title = title;
+    var favorites = 0;
+  }
+  addSong(){
+    const songs = [];
+    songs.push(this.title);
```

Comment:
this is going to push the playlist title. addSong should take an argument `song` and that should be pushed to `this.songs`.